src/client/Microsoft.Identity.Client/Platforms/Features/DesktopOS/Kerberos/AcceptContextFlag.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.Identity.Client.Kerberos.Win32
+namespace Microsoft.Identity.Client.Platforms.Features.DesktopOs.Kerberos
```

Comment:
@yoko-ms - I moved all the low level pinvoke stuff into a directory that is available only on Win32. This will avoid having all the `#if !ANDROID && ...` flags.

Reply:
Thanks for all your works.